### DISCUSSION TITLE
Unable to draw dashed lines

### DISCUSSION THREAD

**Discussion by udavdasha** [cduck/drawSvg#125] [Q&A] [answered]
Hi, and thank you very much for this cool package which I frequently use in my research!

However, I am stuck with the simple thing: how do I draw a dashed line with the Path object? I did not find it in the documentation :(

This works cool:
`p = drawsvg.Path(stroke_width = stroke_width, stroke = stroke, fill = fill, d = d, id = obj_id)  `

But now I would be happy to do something like this, which fails:
`p = drawsvg.Path(stroke_width = stroke_width, stroke_dasharray = stroke_dasharray, stroke_dashoffset = stroke_dashoffset, stroke = stroke, fill = fill, d = d, id = obj_id)`

Thank you very much for the responce.
Daria

**Comment by cduck** [comment, marked as answer]
You have the keyword arguments right but maybe you didn't specify the dash array as a string like `'2,5,3,6'`.  Drawsvg does not automatically translate a Python array for you (yet) into an SVG attribute.
Documentation:  https://cduck.github.io/drawsvg/#stroke_dasharray

Remember you can refer to [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray) for all possible SVG attributes that may not be documented here.

I'm glad you find it useful for research!  If you haven't seen it, my package [latextools](https://github.com/cduck/latextools) has some handy tools for using drawsvg and LaTeX together.
